input.go

```diff
@@ -234,6 +234,18 @@ func IsStandardGamepadLayoutAvailable(id GamepadID) bool {
 	return uiDriver().Input().IsStandardGamepadLayoutAvailable(id)
 }
 
+// UpdateStandardGamepadLayoutMappings updates the gamepad layout definitions using a set of definitions in SDL_GameControllerDB format.
+//
+// Returns whether the mappings were applied, and whether an error occurred parsing the mappings.
+//
+// On platforms where gamepad mappings are not managed by ebiten, this always returns false, nil.
+//
+// UpdateStandardGamepadLayoutMappings must be called on the main thread before ebiten.Run, and is concurrent-safe after ebiten.Run.
```

Comment:
Update this comment.

Reply:
Yes; now it's correct: before ebiten.Run it must be the main thread; after the game loop is running, it's concurrent-safe.

Reply:
Thanks! Perhaps the other functions might be in the same situation, though their comments say just 'concurrent-safe'. I'll revisit them...